lpa-submissions-e2e-tests/cypress/support/common/goToPage.js

```diff
@@ -0,0 +1,8 @@
+/// <reference types = "Cypress"/>
+import defaultPathId from '../../utils/defaultPathId';
+
+module.exports = (page = '', subdirectory = defaultPathId) => {
+  const path = `/${subdirectory}/${page}`;
+  cy.visit(path, {failOnStatusCode:false});
+  cy.checkPageA11y(path);
```

Comment:
I think this should be removed from the go to as it will really slow things down, instead it should be on a per page basis as part of a then check

Reply:
Done.